.github/workflows/CI_build.yml

```diff
@@ -257,16 +257,11 @@ jobs:
         run: |
           export BUILDDIR=$RUNNER_TEMP/matchms/_build/noarch/
           [ "$RUNNING_OS" = "Windows" ] && export BUILDDIR=$RUNNER_TEMP\\matchms\\_build\\noarch\\
-          conda install \
-            --channel bioconda \
-            --channel conda-forge \
-            --channel nlesc \
-            matchms
           # Install matchms without nlesc channel to prevent package to be installed from nlesc channel instead of wanted $BUILDDIR channel
           conda install \
-            --channel bioconda \
+            --channel $BUILDDIR \
             --channel conda-forge \
-            --channel $BUILDDIR -v \
+            --channel bioconda -v \
```

Comment:
I here want to install the package that the workflow builds locally. `conda install` gives priority to the channels based on their order, so when bioconda was before `$BUILDDIR` it would simply pick matchms from bioconda instead of the local version (which would than miss a `save_to_msp` function).

Reply:
I understand - makes sense!